### DISCUSSION TITLE
Comments at start of operation

### DISCUSSION THREAD

**Discussion by cyber-fly** [TimPaterson/Fusion360-Batch-Post#18] [Ideas]
Hello, 
first congratulations on the project. If possible, would you be able to leave comments at the beginning of each operation after joining all operations? This helps a lot to identify which tool will be changed on break. As it is, only the first is left.  Thanks.

![exemple](https://user-images.githubusercontent.com/7515617/124286784-abf27a80-db25-11eb-9d71-eaf344dad304.JPG)

**Comment by TimPaterson** [comment]
What post processor are you using?

**Reply by cyber-fly** [reply]
I'm using Mach3Mill. Thanks

**Comment by TimPaterson** [comment]
An update is now available that copies all comment lines to the final G-code file.

**Reply by cyber-fly** [reply]
Thank you very much, now it's perfect!
